fhir-server-test/src/test/java/com/ibm/fhir/server/test/ServerSpecTest.java

```diff
@@ -423,13 +423,13 @@ public void testReadMedicationAdministrationErrorNotFound() {
     }
 
     // Test: retrieve invalid resource type.
-    @Test(groups = { "server-spec" })
-    public void testReadErrorInvalidResourceType() {
-        WebTarget target = getWebTarget();
-        Response response = target.path("BogusResourceType/1").request(FHIRMediaType.APPLICATION_FHIR_JSON).get();
-        assertResponse(response, Response.Status.BAD_REQUEST.getStatusCode());
-        assertExceptionOperationOutcome(response.readEntity(OperationOutcome.class), "The virtual resource types feature is not enabled for this server");
-    }
+//    @Test(groups = { "server-spec" })
+//    public void testReadErrorInvalidResourceType() {
+//        WebTarget target = getWebTarget();
+//        Response response = target.path("BogusResourceType/1").request(FHIRMediaType.APPLICATION_FHIR_JSON).get();
+//        assertResponse(response, Response.Status.BAD_REQUEST.getStatusCode());
+//        assertExceptionOperationOutcome(response.readEntity(OperationOutcome.class), "The virtual resource types feature is not enabled for this server");
+//    }
```

Comment:
I commented this out for now, should get removed as part of https://github.com/IBM/FHIR/issues/153